backend/models/post_models.py

```diff
@@ -0,0 +1,24 @@
+from enum import Enum
+
+from pydantic import BaseModel
+
+
+class Post(BaseModel):
+    """Model for post metadata"""
+
+    title: str
```

Comment:
can we add field validator here? https://docs.pydantic.dev/latest/concepts/validators/#field-validators
i.e. "has to be at least 5 characters?"